### ISSUE TITLE
OTPP ADE <va-link> not using correct aria-label prop

### ISSUE THREAD

**Issue by jtmst** [department-of-veterans-affairs/va.gov-team#120025]
#### **Issue: \<va-link\> not using correct aria-label prop**

* **Page:** Current copay balances, Current debts  
* **First observed:** MacOS \+ Chrome  
* **Expected 508 Office score:** Critical  
* **Expected CC severity level:** 2  
* **Expected CC experience standard:** User knows where a button or link leads  
* **Expected WCAG violation:** 1.3.1 Info and Relationships  
* **Description:** “Review details,” “Resolve this bill,” and “Resolve this debt” links aren’t using the correct prop to generate an aria-label. The aria-label isn’t rendering properly as a result.  
* **How to recreate:** Inspect the source on those links and check the accessibility panel, note the “name”:
  <img width="631" height="159" alt="Image" src="https://github.com/user-attachments/assets/bef49fa5-97d6-451c-ab9f-4f74e391f97f" />
* **Recommendation:** In any \<va-link\>, use ‘label’ instead of ‘aria-label’ (note the [prop list in Storybook](https://design.va.gov/storybook/?path=/docs/components-va-link--docs))        

## Tasks
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3          

## Design and Front End sync on implementation of design
- [ ] Required
- [ ] Complete
- [ ] Not Required           

## Acceptance Criteria
- [ ] Item one from the description performs as described in the description
- [ ] Item two from the description performs as described in the description      

## Testing
- [ ] N/A if non development work
- [ ] Testing passed and documented in this ticket based off the "Then" statement in the description     

Test procedures:      
*If additional testing steps or credentials to perform testing are needed list them here*    

## Bug Ticket Resolution  
- [ ] If this is a Bug or Fix - Check this box and complete the information below.
- Additional description of bug once the cause was identified.  
      - Answer
- Root Cause - what caused this issue?  
      - Answer
- Impact - how were Veterans impacted? (i.e. FSR submissions which ones, date range, etc)  
      - Answer
- Resolution Details - what was the fix & when did it go live.  
      - Answer
- Follow Up Actions - any additional/further work that came out of fixing the bug?  
      - Answer
- Preventative Measures - any planned steps to prevent similar issues in the future?  
      - Answer

</details>
